pygdtf/__init__.py

```diff
@@ -104,6 +104,12 @@ def _read_xml(self):
         model_collect = self._root.find('Models')
         if model_collect:
             self.models = [Model(xml_node=i) for i in model_collect.findall('Model')]
+        for model in self.models:
```

Comment:
The code inside `pygdtf` comes from [this repo](https://github.com/jackdpage/python-gdtf) by @jackdpage.
If you could also open a PR there explaining your changes, it'd be very cool so we can keep the versions synced. (Also it might be useful for other projects)

Reply:
As for pymvr, this is my code, it is very basic at the moment. I added a DEPENDENCIES.md file to track the dependencies.

I know about pygdtf, i have done several changes. The original library is GDTF 1.1 related, while there is already GDTF 1.2. Some things like LogicalChannel Default values are different. I would also like to move the list of channel parsing into the library. Not sure when i find the time but yes, making an upstream PR is on my list...